GOVERNANCE.md

```diff
@@ -1,88 +1,234 @@
+## Removing project areas
 
-# When does a maintainer lose maintainer status
+Project areas are removed by removing all maintainers for that area and removing the corresponding team from the Backstage GitHub organization. The project area can be re-added later if there is a need for it. Reasons for removal may include lack of activity, lack of maintainers, or lack of relevance to the project.
 
-If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be moved to emeritus status. In extreme cases this can also occur by a vote of the sponsors and maintainers per the voting process below.
+# Project Roles
 
-# End User Sponsors
+## Contributor
 
-## Role of a Backstage End User Sponsor
+A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
 
-- Provide support for Backstage by removing blockers, securing funding, providing advocacy, feedback, and ensuring project continuity and long term success.
-- Assist Backstage maintainers in prioritizing upcoming roadmap items and planned work.
-- Provide neutral mediation for any disputes that arise as part of the project.
+### Responsibilities
 
-## Backstage End User Sponsor Membership
+- Follow the [CNCF CoC](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
+- Follow the project [contributing guide](CONTRIBUTING.md)
+
+### How to get involved
+
+- Participating in community discussions
+- Helping other users
+- Submitting bug reports
+- Commenting on issues
+- Trying out new releases
+- Attending community events
+
+### How to contribute
+
+- Report and sometimes resolve issues
+- Occasionally submit PRs
+- Contribute to the documentation
+- Show up at meetings, take notes
+- Answer questions from other community members
+- Submit feedback on issues and PRs
+- Test releases and patches and submit reviews
+- Run or help run events
+- Promote the project in public
+
+## Organization Member
+
+An org member is a frequent contributor that has become a member of the Backstage GitHub organization. In addition to the responsibilities of contributors, an org member is also expected to be reasonably active in the community through continuous contributions of any type.
+
+An Organization Member must meet the responsibilities and has the requirements of a Contributor.
+
+### Responsibilities
+
+- Continues to contribute regularly, as demonstrated by having at least 10 GitHub contributions per year, as found in [Devstats](https://backstage.devstats.cncf.io/d/48/users-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=All&from=now-1y&to=now&var-users=All).
```

Comment:
There are quite a few other types of contributions that aren't captured in Devstats - running meetings / meetups, for example, so I'd add a bit to the end, something like, "or contributions of a similar effort that might not be captured in Devstats."

Reply:
Good call, thanks! 👍